### PR TITLE
Sledgehammer Audit

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Balance "Sledgehammer Audit"

#### Purpose of change

Sledgehammers weren't durable, didn't have an accurate to-hit, and the technique list for the shorter ones didn't make sense.

#### Describe the solution

Gives all the sledges the durability flag, gives the regular ones a "short" length as you grip a sledgehammer near the head on a swing, and gave the short ones a "hand" length, removed their techniques, with the short sledge having "clumsy" balance and the engineers having "uneven"

#### Describe alternatives you've considered

Not doing this

#### Testing
Tested locally, works fine
image

### CODE REVIEW

**Comment by Kantonine** [comment]
> #### Summary
> 
> Balance "Sledgehammer Audit"
> #### Purpose of change
> 
> Sledgehammers weren't durable, didn't have an accurate to-hit, and the technique list for the shorter ones didn't make sense.
> #### Describe the solution
> 
> Gives all the sledges the durability flag, gives the regular ones a "short" length as you grip a sledgehammer near the head on a swing, and gave the short ones a "hand" length, removed their techniques, with the short sledge having "clumsy" balance and the engineers having "uneven"
> #### Describe alternatives you've considered
> 
> Not doing this
> #### Testing
> 
> Tested locally, works fine image
> #### Additional context

Wrong Screenshot!

**Comment by Holli-Git** [comment]
> Wrong Screenshot!

Just an old one. Fixed

File: data/json/items/melee/bludgeons.json [hunk 1]
```diff
@@ -814,9 +813,9 @@
     "symbol": "/",
     "material": [ "wood", "steel" ],
     "looks_like": "hammer_sledge",
-    "flags": [ "NONCONDUCTIVE", "ALWAYS_TWOHAND" ],
+    "flags": [ "DURABLE_MELEE", "NONCONDUCTIVE", "ALWAYS_TWOHAND" ],
```
Comment (Maleclypse) [inline data/json/items/melee/bludgeons.json:816]:
```suggestion
    "flags": [  "NONCONDUCTIVE", "ALWAYS_TWOHAND" ],
```
Sledges aren’t durable in melee. I’ve thrown too many heads. Unless this is a one piece cast metal sledge but then it loses non-conductive

Comment (Holli-Git) [reply data/json/items/melee/bludgeons.json:816]: I think bashing solid objects wears a tool down much more than hitting relatively soft targets. If I went to town on a bunch of concrete slabs with a mace I think it'd wear down in a similar time

**Pull request merged**